main.go

```diff
@@ -103,6 +105,33 @@ func main() {
 		log.Errorf("Failed to setup logger reason: %s", err)
 	}
 
+	if bot.config.NTPClient.Level != -1 {
+		bot.config.CheckNTPConfig()
+		NTPTime, errNTP := ntpclient.NTPClient(bot.config.NTPClient.Pool)
+		currentTime := time.Now()
+		if errNTP != nil {
+			log.Warnf("NTPClient failed to create: %v", errNTP)
+		} else {
+			NTPcurrentTimeDifference := NTPTime.Sub(currentTime)
+			if bot.config.NTPClient.Level == 0 {
+				configTime := *bot.config.NTPClient.AllowedDifference
+				configNegativeTime := *bot.config.NTPClient.AllowedNegativeDifference
+				log.Debugln(configTime)
+				log.Debugln(configNegativeTime)
+				if NTPcurrentTimeDifference > configTime || NTPcurrentTimeDifference < configNegativeTime {
+					log.Warnf("Time out of sync (NTP): %v | (time.Now()): %v | (Difference): %v | (Allowed): +%v / %v", NTPTime, currentTime, NTPcurrentTimeDifference, bot.config.NTPClient.AllowedDifference, bot.config.NTPClient.AllowedNegativeDifference)
+					disable, errNTP := bot.config.DisableNTPCheck(os.Stdin)
+					if errNTP != nil {
+						log.Errorf("failed to disable ntp time check reason: %v", err)
+					}
+					log.Info(disable)
+				}
+			} else {
+				log.Warnf("Time out of sync (NTP): %v | (time.Now()): %v | (Difference): %v | (Allowed): +%v / %v", NTPTime, currentTime, NTPcurrentTimeDifference, bot.config.NTPClient.AllowedDifference, bot.config.NTPClient.AllowedNegativeDifference)
```

Comment:
`(Allowed): +%v / %v"` -> `(Allowed): +%v / -%v"`

Reply:
This value is now a negative number and will be displayed as a negative value